content/spin/http-trigger.md

```diff
@@ -92,45 +119,186 @@ will be handled by `component-1`:
 trigger = { type = "http", base = "/"}
 
 [[component]]
-id = "component-1"
+id = "user-manager"
 [component.trigger]
-route = "/foo/..."
+route = "/users/..."
 
 [[component]]
-id = "component-2"
+id = "admin"
+[component.trigger]
+route = "/users/admin"
+
+[[component]]
+id = "shop"
 [component.trigger]
 route = "/..."
 ```
 
-> Note: Although the route defined by `component-2` also matches `/foo/bar`, the route 
-> defined by `component-1` has the longer matching prefix, i.e. `/foo/`.
+### Health Check Route
 
-Every HTTP application has a special route always configured at `/.well-known/spin/health`, which
+Every HTTP application automatically has a special route always configured at `/.well-known/spin/health`, which
 returns `OK 200` when the Spin instance is healthy.
 
-Once Spin selects a component to handle an incoming request based on the route
-configuration, it will instantiate and execute that component based on its
-defined _HTTP executor_, and the next sections explore the two ways of building
-HTTP components based on the two available executors.
+## HTTP Components
+
+> Spin has two ways of running HTTP components, depending on language support for the evolving WebAssembly component standards.  This section describes the default way, which is currently used by Rust, JavaScript/TypeScript, Python, and TinyGo components.  For other languages, see [HTTP Components using Wagi](#http-components-using-wagi) below.
```

Comment:
Hi @karthik2804 
This all looks fine and passes `bart check`, any idea what this specific error in the `bart check` is referring to?
```
Error rendering markdown: Error rendering "Unnamed template" line 144, col 1: Helper not defined: "tabs"
```

```bash
tpmccallum@Tims-MacBook-Pro-3 developer % bart check content/spin/http-trigger.md 
Error rendering markdown: Error rendering "Unnamed template" line 144, col 1: Helper not defined: "tabs"
✅ content/spin/http-trigger.md
```

Reply:
Yea that is because `bart` does not know where to find the `shortcodes`. I will PR a fix to te CI.